### PR TITLE
Profile page initial template

### PR DESCRIPTION
**Pull request by michael-caira**
@leandrabudau2939 and I, Implemented the profile page figma: https://www.figma.com/file/LLZiwB6naD3qtkoG6seG27/Image-Repo-UI?node-id=8%3A51

Everything is hardcoded as specified for now, but the template should be good to go for connecting to real data.

One thing we may need is a modal for uploading a new user profile pic, but we could just do something basic/not do a drag and drop type of modal.

Let us know what you think.

### CODE REVIEW

**Comment by Mwpereira** [comment]
Currently, there is no functionality for this button?

image

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import './App.css';
 
 import { Button, useColorMode } from '@chakra-ui/react';
-import { BrowserRouter as Router, Navigate,Route, Routes } from 'react-router-dom';
+import { BrowserRouter as Router, Navigate, Route, Routes } from 'react-router-dom';
 
 import Dashboard from './pages/Dashboard';
 // import Container from './components/Container';
```
Comment (jamestang12) [inline client/src/App.tsx:7]: Please remove this if we are not using it

**Review by jamestang12 — COMMENTED** [review]
Overall LGTM, please remove unnecessary import and fix the styling issue mentioned by @Mwpereira 

Great work 👍

**Comment by michael-caira** [comment]
Good catches, added functionality for image selection, removed unused imports, and added more spacing, let me know if you guys think it's good.

Tried to keep the spacing consistent with the spacing at the top between the user handle and the beginning of the form.
Screenshot 2022-11-06 at 10 42 05 PM

**Review by jamestang12 — APPROVED** [review]
LGTM

**Pull request merged**